tests/accel/upgrade/internal/createres/create.go

```diff
@@ -0,0 +1,95 @@
+package createres
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/deployment"
+	"github.com/openshift-kni/eco-goinfra/pkg/route"
+	"github.com/openshift-kni/eco-goinfra/pkg/service"
+
+	upgradeinittools "github.com/openshift-kni/eco-gotests/tests/accel/internal/accelinittools"
+	"github.com/openshift-kni/eco-gotests/tests/accel/upgrade/internal/upgradeparams"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// Workload Create a workload with test image.
+func Workload(apiClient *clients.Settings, workloadImage string) (*deployment.Builder, error) {
+	glog.V(90).Infof("Creating Deployment %q", upgradeparams.DeploymentName)
+
+	workloadDeployment, err := deployment.NewBuilder(
+		upgradeinittools.HubAPIClient, upgradeparams.DeploymentName, upgradeparams.TestNamespaceName, map[string]string{
+			"app": upgradeparams.DeploymentName,
+		}, &corev1.Container{
```

Comment:
We have container struct. Please create container using
pod.NewContainerBuilder package.